### PR TITLE
Improve user experience when generating new app

### PR DESCRIPTION
**Pull request by sundaycrafts**
My PR is a:

💅 Enhancement

Main update on the:

Framework

It is a tiny and proposal pull request. Please give me some advice 🙏

I added a default value to auth DB in generating new app.

I think most users use "admin" DB as an auth DB.
But it may cause confusion to the user unfamiliar for Mongo DB if we don't offer a default value.

So I think it is more good for user experience that suggesting it as a default value.

### CODE REVIEW

**Comment by maturanomx** [comment]
🤔 In my experience I have not to specify an authentication user for connect to mongo. This just installing mongo from repository without further modifications.

**Comment by sundaycrafts** [comment]
Oh, I see. I used MongoDB only via docker container or some IaaS like mlab. So I didn't know MongoDB can use without auth😲.

And I had realized if it has default auth value the user who doesn't need to authenticate will confuse.
So I withdraw this pull request.

@maturanomx Thank you awesome advice!

**Comment by sundaycrafts** [comment]
By the way, how about it just adding a hint?

File: packages/strapi-generate-new/lib/before.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ module.exports = (scope, cb) => {
                   when: !hasDatabaseConfig && scope.client.database === 'mongo',
                   type: 'input',
                   name: 'authenticationDatabase',
-                  message: 'Authentication database:',
+                  message: 'Authentication database(Maybe "admin" or blank):',
```
Comment (maturanomx) [inline packages/strapi-generate-new/lib/before.js]: Space between "database" and "(Maybe..."...

**Comment by maturanomx** [comment]
If a hint helps to avoid a common error, go ahead.

I just point a missing space in the message :)

**Comment by sundaycrafts** [comment]
Thank you! I fixed it!

**Comment by Aurelsicoko** [comment]
LGTM 👍 

Thank you @sundaycrafts!

**Pull request merged**